modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/Direction.scala

```diff
@@ -0,0 +1,11 @@
+/** Rosetta Build Once. Integrate Your Blockchain Everywhere.
+  *
+  * The version of the OpenAPI document: 1.4.12 Contact: team@openapitools.org
+  *
+  * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech). https://openapi-generator.tech
+  */
+
+package org.tessellation.rosetta.server.model
+
+case class Direction(
```

Comment:
Empty case class?

Reply:
Some of these fail to generate properly, but we should still preserve them in case we need a manual modification later.